sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -618,6 +618,15 @@ AllocaCommandBase *Scheduler::GraphBuilder::getOrCreateAllocaForReq(
         } else {
           LinkedAllocaCmd->MIsActive = false;
           Record->MCurContext = Queue->getContextImplPtr();
+
+          std::set<Command *> Deps =
```

Comment:
```suggestion
          auto Deps =
```

Reply:
I believe this change is not aligned with [codestyle guidelines](https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable).

Reply:
yes that true, i am too use to auto everything!